ynput/OpenPype
# Push to project tool: Prepare push to project tool for AYON

## Changelog Description
Cloned Push to project tool for AYON and modified it.

## Additional info
The tool is converted to point that it works, it's not full AYON conversion. The conversion was mainly using new widgets and small refactor of controllers. The integration logic stayed as was which means the integration is still guessing the data based on conversion utils used in operations.

## Testing notes:
1. Make sure you have at least one library project
2. Open AYON launcher
3. Open `Loader` from tray
4. Go to a different project
5. Choose a product of your choice
6. Run action Push to library on the product
7. Go through push to project UI
8. Validate everything works as expected
